packages/lists/README.md

```diff
@@ -0,0 +1,17 @@
+# [@uifabric/lists](http://dev.microsoft.com/fabric)
+
+##### Lists components for Office UI Fabric React
```

Comment:
Can you update the readme with more or less what you put in the PR description?

Reply:
Important point! Added a blurb in https://github.com/OfficeDev/office-ui-fabric-react/pull/9106/commits/54e06031518e17c1767e6250143ab760cdf1be53. Will revise as code takes shape.